tests/integration/test_yarn.py

```diff
@@ -45,7 +45,7 @@ def test_yarn_packages(
     request: pytest.FixtureRequest,
 ) -> None:
     """
-    Test fetched dependencies for yarn 2+.
```

Comment:
nitpick: I would change this to 'v3' explicitly as who knows how many changes will be necessary for v4. If the test invocation can remain unchained, then I say we convert the string to 'berry'.

Reply:
I think the test invocation will remain unchanged, but we'll need to change the ref and the test output data. Another option is to leave it simply at `yarn` :man_shrugging: